### PR TITLE
Update volume expansion with better kubectl commands

### PR DESCRIPTION
**Pull request by sdodsley**
Better to use `kubectl patch` that `kubectl edit`.
Also:
* correct the markdown formatting
* make the comment about pod restart more prominent
* remove redundant YAML text

### CODE REVIEW

File: docs/csi-volume-expansion.md [hunk 1]
```diff
@@ -167,5 +175,17 @@ kubectl apply -f https://raw.githubusercontent.com/purestorage/helm-charts/maste
...
-kubectl edit pvc pure-claim-file
+kubectl patch pvc pure-claim-file -p='{"spec": {"resources": {"requests": {"storage": "20Gi"}}}}'
+```
+
+Check that both the PV and PVC are immeadiately expanded. No pod restart is required.
+
+```bash
+# kubectl get pvc pure-claim-file
+NAME               STATUS    VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+pure-claim-file    Bound     pvc-2ba56b33-3412-2965-f4e4-983de21ba772   10Gi       RWO            pure-file      25s
```
Comment (pure-jliao) [inline docs/csi-volume-expansion.md:186]: Here the size should be 20?

File: docs/csi-volume-expansion.md [hunk 2]
```diff
@@ -167,5 +175,17 @@ kubectl apply -f https://raw.githubusercontent.com/purestorage/helm-charts/maste
...
+# kubectl get pv
+NAME                                       CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS    CLAIM                      STORAGECLASS   REASON    AGE
+pvc-2ba56b33-3412-2965-f4e4-983de21ba772   20Gi       RWX            Delete           Bound     default/pure-claim-file    pure-file                27s
```
Comment (pure-jliao) [inline docs/csi-volume-expansion.md:189]: Access mode should be RWO?

File: docs/csi-volume-expansion.md (hunk 1 above)
Comment (sdodsley) [reply docs/csi-volume-expansion.md:186]: Yup - typo

File: docs/csi-volume-expansion.md (hunk 2 above)
Comment (sdodsley) [reply docs/csi-volume-expansion.md:189]: `pure-file` is really only RWX. Are you saying that RWX cannot be expanded? Even RWX on raw block?

Comment (pure-jliao) [reply docs/csi-volume-expansion.md:189]: Oh didn’t know. I mean the pvc is RWO but the PV is RWX?

Comment (sdodsley) [reply docs/csi-volume-expansion.md]: Made it RWO, but let's confirm about RWX expansion support on `pure-file` and raw block in `pure-block`.

Comment (pure-jliao) [reply docs/csi-volume-expansion.md]: Confirmed it should work for both.

File: docs/csi-volume-expansion.md (hunk 1 above)
Comment (pure-jliao) [reply docs/csi-volume-expansion.md:186]: here the issue is not resolved.

Comment (caryli-ps) [reply docs/csi-volume-expansion.md:186]: Line 186 CAPACITY needs to be updated to show `20Gi` as well.

**Pull request merged**